tests/e2e/lib.sh

```diff
@@ -276,7 +276,20 @@ deploy_sensor_via_operator() {
         --output-secrets - \
     | kubectl -n stackrox apply -f -
 
-    kubectl apply -n stackrox -f tests/e2e/yaml/secured-cluster-cr.yaml
+    if [[ -n "${COLLECTION_METHOD:-}" ]]; then
+	echo "Overriding the product default collection method due to COLLECTION_METHOD variable: ${COLLECTION_METHOD}"
+    else
+       die "COLLECTION_METHOD not set"
+    fi
+
+    upper_case_collection_method="$(echo "$COLLECTION_METHOD" | tr '[:lower:]' '[:upper:]')"
+    env - \
+      collection_method="$upper_case_collection_method" \
+    envsubst \
+      < tests/e2e/yaml/secured-cluster-cr.envsubst.yaml \
+      > /tmp/secured-cluster-cr.yaml
+
+    kubectl apply -n stackrox -f /tmp/secured-cluster-cr.yaml
```

Comment:
[nit] Writing to just `/tmp` although should work in 95%+ of cases is not elegant, prone to failure if `/tmp` is fully used or absent, and simply redundant.

Why not just pipe directly without creating any files?

```bash
envsubst ... | kubectl apply -n stackrox -f -
```

If you don't mind of such change, I'd ask you to also update the similar place in `deploy_central_via_operator()` so that we don't have two places in this file that do the same thing slightly differently.

Reply:
Done